doc/tutorials/sandbox_config.ipynb

```diff
@@ -0,0 +1,121 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #1.    #TODO: use the proper secret store.
</pre><p>I would still suggest removing this line and creating a ticket for changing to the secret store in the notebooks instead. We generally try not to have "todo"s in our merges to main. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/exasol/data-science-sandbox/pull/22/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>